### PR TITLE
Adds 3 Column Footer Contact Form

### PR DESCRIPTION
**Pull request by nitrokevin**
3 column footer block

### CODE REVIEW

**Comment by nitrokevin** [comment]
Sorry If I've done this wrong, first building block and first time GitHub user !

**Comment by IamManchanda** [comment]
Hey congrats for your first PR... Will take a look soon!

**Comment by IamManchanda** [comment]
Hi @nitrokevin Before i review your PR you first need to correct the indentation of your code (2 spaces)
For your reference => https://github.com/zurb/building-blocks/pull/188/files

**Comment by nitrokevin** [comment]
Thanks! I hadn't even realised my editor was putting 4 spaces until you mentioned it, on it now

File: src/building-blocks/3-column-footer-contact-form/3-column-footer-contact-form.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  - footer
+category: containers
+description: A Responsive footer with contact details, newsletter signup, contact form and company details. Each column aligns to the bottom.
+versions:
+  - 6.3.1
+  - 6.3.0
+author:
+  github: nitrokevin
+  name: Kevin Neal
+grid: float
```
Comment (IamManchanda) [inline src/building-blocks/3-column-footer-contact-form/3-column-footer-contact-form.yml]: Remove this line 
BTW , we are working on this => https://github.com/zurb/building-blocks/issues/211

File: src/building-blocks/engadget-footer/engadget-footer.scss [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      @include grid-row();
+      color:$light-gray;
+      padding: rem-calc(60) 0;
+      a {
+        color:$light-gray !important;
+        }
+	  .links {
+  	  @include breakpoint(small) {
+        @include grid-layout(2, '.link-column', $grid-column-gutter);
+        }
```
Comment (IamManchanda) [inline src/building-blocks/engadget-footer/engadget-footer.scss]: Still lot of whitespaces!

File: src/building-blocks/3-column-footer-contact-form/3-column-footer-contact-form.yml (hunk 1 above)
Comment (IamManchanda) [inline src/building-blocks/3-column-footer-contact-form/3-column-footer-contact-form.yml]: Oh no sorry .... my mistake i was not aware of this param .... keep it !

File: src/building-blocks/engadget-footer/engadget-footer.scss (hunk 2 above)
Comment (IamManchanda) [inline src/building-blocks/engadget-footer/engadget-footer.scss]: Awaiting revert! @nitrokevin

**Comment by rafibomb** [comment]
I made the updates - merged!
Well done @nitrokevin!
Good feedback @IamManchanda !

**Pull request merged**